src/Client/SharedComponents/Metadata/Forms.fs

```diff
@@ -286,38 +286,126 @@ module private Helper =
                 ]
             ]
         ]
-        
+
+open JsBindings
 
 type FormComponents =
 
     [<ReactComponent>]
-    static member InputSequence<'A>(inputs: ResizeArray<'A>, constructor: unit -> 'A, setter: ResizeArray<'A> -> unit, inputComponent: 'A * ('A -> unit) * (MouseEvent -> unit) -> ReactElement, ?label: string) =
-        Bulma.field.div [
-            if label.IsSome then Bulma.label label.Value
-            Bulma.field.div [
-                Html.orderedList [
-                    prop.className "grid grid-cols-1 gap-2"
+    static member InputSequenceElement(key: string, id: string, listComponent: ReactElement) =
```

Comment:
You have the parameter key that is never used in the method

Reply:
that is correct! it is not used in the method, but when transpiled to jsx this is similiar to

```tsx
<InputSequenceElement key={key} />
```

Which is the react required way keep react states on possible reorder events. [more](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)

This looks strange but it is not necessary to use key inside the component, as simply passing the key parameter already sets key for react.

I actually only found out about this by looking into the Feliz.React source code. I am working on a PR to expand the Feliz docs accordingly